sentry-ruby/spec/sentry/transaction_spec.rb

```diff
@@ -47,11 +47,28 @@
       end
     end
 
-    context "when tracing is disabled" do
+    context "when tracing is enabled (= 0)" do
       before do
         configuration.traces_sample_rate = 0.0
       end
 
+      it "returns correctly-formatted value" do
+        child_transaction = described_class.from_sentry_trace(sentry_trace, op: "child")
+
+        expect(child_transaction.trace_id).to eq(subject.trace_id)
+        expect(child_transaction.parent_span_id).to eq(subject.span_id)
+        expect(child_transaction.parent_sampled).to eq(true)
+        # doesn't set the sampled value
+        expect(child_transaction.sampled).to eq(nil)
+        expect(child_transaction.op).to eq("child")
+      end
+    end
+
+    context "when tracing is disabled" do
+      before do
+        configuration.traces_sample_rate = false
```

Comment:
This value should typically be numeric or `nil`. I suspect the default value is `nil`? If so I'd use that here.